### PR TITLE
storage: fix "size" property

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

**Comment by brauner** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
@brauner lxd/storage_dir.go:81: multiple-value shared.PathIsEmpty() in single-value context

**Comment by brauner** [comment]
@stgraber, updated, sorry.

File: lxd/patches.go [hunk 1]
```diff
@@ -2321,6 +2322,72 @@ func patchStorageZFSnoauto(name string, d *Daemon) error {
...
+			// It shouldn't be possible that false volume types
+			// exist in the db, so it's safe to ignore the error.
+			volumeType, _ := storagePoolVolumeTypeNameToType(volume.Type)
+			// Update the volume config.
+			err = db.StoragePoolVolumeUpdate(d.db, volume.Name,
+				volumeType, poolID, volume.Description,
+				volume.Config)
+			if err != nil {
+				return err
+			}
```
Comment (stgraber) [inline lxd/patches.go]: Do we need any logic to remove the quota from the zfs dataset itself too?

**Pull request merged**